_security-analytics/sec-analytics-config/log-types.md

```diff
@@ -8,22 +8,44 @@ nav_order: 16
-| **Microsoft 365 logs** | Microsoft 365 audit logs collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity.
-| **Okta events** | These logs record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. 
-| **Microsoft Azure logs** | Logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services.
-| **VPC Flow logs** | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) capture information about the IP traffic going to and from network interfaces in your VPC.
+Logs contain raw data about events that happen throughout a system and in its separate parts. As of OpenSearch 2.11, log types are grouped by category to help select, filter, and search the log types. 
+
+To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, it's category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected, and where you can filter the list by the category. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- |
+| Access Management | `Ad_ldap` | Active Directory logs that track such things as LDAP queries, errors from the LDAP server, time-out events, and unsecure LDAP binds. |
+| Access Management | `Apache_access` | A log type that is responsible for recording data for all requests processed by an Apache HTTP server. |
+| Access Management | `Okta` | These logs record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. |
+| Applications | `GitHub` | Logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). |
+| Applications| `Gworkspace` | Logs for Google Workspace that can monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |  
+| Applications| `M365` | Microsoft 365 audit logs collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity. |
+| Cloud Services | `Azure` | Logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services. |
```

Comment:
Azure isnt descriptive. Can we rename this appropriately? Whats the exact log type we are monitoring?